src/main/java/io/github/syst3ms/skriptparser/effects/EffAsk.java

```diff
@@ -0,0 +1,54 @@
+package io.github.syst3ms.skriptparser.effects;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.expressions.ExprAnswer;
+import io.github.syst3ms.skriptparser.lang.Effect;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.TaggedExpression;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+import java.util.Scanner;
+
+/**
+ * Ask in the system console a specific message and wait for the answer of the user.
+ * The code in the section will be executed once the user answer or enter any value.
+ * The code after the section will run even if the user haven't answered yet.
+ *
+ * @name Ask
+ * @type EFFECT
+ * @pattern ask for %string%
+ * @since ALPHA
+ * @author ItsTheSky
+ */
+public class EffAsk extends Effect {
+
+    static {
+        Parser.getMainRegistration().addEffect(
+                EffAsk.class,
+                "ask for %string%"
```

Comment:
```suggestion
                "(ask [for]|input) %string%"
```
Not sure how you feel about this syntax. I think the input function is too well-known to not include it in the pattern.

Reply:
Yeah I love it that way too